src/main/java/core/mate/academy/service/impl/TruckProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Truck;
+import core.mate.academy.service.MachineProducer;
+
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        return List.of(new Truck());

```

Comment:
The `get` method should return a list containing 2-3 `Truck` instances as per the task requirements. Consider creating additional `Truck` objects and adding them to the list.